neo/public/stylesheets/main.css

```diff
@@ -119,7 +113,7 @@ search-bar {
 #powerby{
            position: absolute;
            bottom: 0%;
-           left: 71%;
+           left: 70%;
```

Comment:
I assume it can easily shown the pros and cons by changing this number, right? then let's decide during the meeting tomorrow.